src/components/timeline/TrackMemoryGraph.js

```diff
@@ -134,6 +135,7 @@ class TrackMemoryCanvas extends React.PureComponent<CanvasProps> {
           deviceLineHalfWidth;
         if (i === 0) {
           // This is the first iteration, only move the line.
```

Comment:
optional nit: that would be:
```suggestion
          // This is the first iteration, only move the pencil, do not draw.
```